### PR TITLE
Wrapper in api

### PR DESCRIPTION
**Pull request by arthurrfoy**

### CODE REVIEW

File: backend/src/controllers/project.ts [hunk 1]
```diff
@@ -5,17 +5,20 @@ import { verifyParameters } from './utils/verifyParams';
...
-	const { userId } = req.params
 	const {
 		name,
+    	userId
 	} = req.body
-	const newProject = await db.createNewProject(parseInt(userId), name)
+	console.log()
+  	if(!verifyParameters(name)) return res.sendStatus(400);
+	const newProject = await db.createNewProject(userId, name)
 	return res.json(newProject)
```
Comment (anlhu) [inline backend/src/controllers/project.ts:15]: Do we need to parseInt here?

File: backend/src/controllers/project_controller.ts [hunk 2]
```diff
@@ -17,8 +18,8 @@ export class ProjectController {
    */
   static async getProjectsFromUser(req: express.Request, res: express.Response) {
       try {
-          const projects = await ProjectModel.getProjectsByUser(req.userId);
-          return res.status(200).json(projects);
+        //   const projects = await ProjectModel.getProjectsByUser(req.userId);
+          return res.status(200);
       } catch (error) {
```
Comment (anlhu) [inline backend/src/controllers/project_controller.ts:23]: Does this run? or just return

File: backend/src/controllers/user.ts [hunk 3]
```diff
@@ -27,25 +27,25 @@ export const login = async (req: express.Request, res: express.Response) => {
...
+};
+// 	const db = Database.getInstance()
+// 	const {
+// 		username,
+// 		email,
+// 		password,
+// 	} = req.body
+// 	const user = await db.updateUser(parseInt(userId), username, email, password, salt, sessionToken)
+// 	return res.json(user)
+// };
```
Comment (anlhu) [inline backend/src/controllers/user.ts]: Was there something wrong with it before?

File: backend/src/index.ts [hunk 4]
```diff
@@ -51,7 +53,12 @@ AppDataSource.initialize().then(async () => {
 
 	/// Start express
 	app.use(express.json())
-  	app.use('/api', router());
+  app.use(cors({
+    origin:"http://localhost:3000",
+    credentials: true,
+  }));
+  app.use(cookieParser());
+  app.use('/api', router());
```
Comment (anlhu) [inline backend/src/index.ts:61]: What is the cors?

File: backend/src/controllers/project.ts (hunk 1 above)
Comment (anlhu) [reply backend/src/controllers/project.ts:15]: nvm, userid is a number now

File: backend/src/controllers/user.ts (hunk 3 above)
Comment (anlhu) [reply backend/src/controllers/user.ts]: It isnt the most urgent API so if its a TODO I think thats fine

**Review by anlhu — APPROVED** [review]

File: backend/src/controllers/project_controller.ts (hunk 2 above)
Comment (arthurrfoy) [reply backend/src/controllers/project_controller.ts:23]: isnt that the other teams file?

Comment (anlhu) [reply backend/src/controllers/project_controller.ts:23]: my bad!

**Pull request merged**